### PR TITLE
(search) normalize api and refactor doc_count where used

### PR DESCRIPTION
**Pull request by oyvindis**
Jira issue link: [link](LINK_HERE)

> check applicable boxes

- [ ] I have made tests to match the user stories
- [ ] This code does not require tests that match user stories

### CODE REVIEW

File: applications/search/src/pages/datasets-report-page/report-stats/__snapshots__/report-stats.component.test.jsx.snap [hunk 1]
```diff
@@ -62,7 +62,7 @@ exports[`should render ReportStats correctly 1`] = `
               <i
                 className="fa fdk-fa-left fa-unlock fdk-color-offentlig"
               />
-              463
+              0
```
Comment (markkopaas) [inline applications/search/src/pages/datasets-report-page/report-stats/__snapshots__/report-stats.component.test.jsx.snap]: dette ser rart ut

File: applications/search/src/pages/search-page/search-publishers-tree/__snapshots__/search-publishers-tree.component.test.jsx.snap [hunk 2]
```diff
@@ -54,7 +54,7 @@ exports[`should render SearchPublishersTree correctly with active filter 1`] = `
               nodeLabel={
                 <FilterOption
                   active={true}
-                  count={91}
+                  count={null}
```
Comment (markkopaas) [inline applications/search/src/pages/search-page/search-publishers-tree/__snapshots__/search-publishers-tree.component.test.jsx.snap]: den ser rart ut

File: applications/search/src/pages/datasets-report-page/report-stats/__snapshots__/report-stats.component.test.jsx.snap (hunk 1 above)
Comment (oyvindis) [reply applications/search/src/pages/datasets-report-page/report-stats/__snapshots__/report-stats.component.test.jsx.snap]: Rettet

File: applications/search/src/pages/search-page/search-publishers-tree/__snapshots__/search-publishers-tree.component.test.jsx.snap (hunk 2 above)
Comment (oyvindis) [reply applications/search/src/pages/search-page/search-publishers-tree/__snapshots__/search-publishers-tree.component.test.jsx.snap]: Rettet

**Pull request merged**